.github/portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="content">
+        <header>
+            <div class="menuBox">
+                <div class="dropdown">
+                    <button class="dropbtn">Dropdown</button>
+                    <div class="dropdown-content">
+                      <a href="bio.html">Bio</a>
+                      <a href="experience.html">Experience</a>
+                      <a href="index.html">Home</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="interestStatementBox">
+                Lorem ipsum dolor sit amet, probo idque postea ea qui,<br>
+                 diam petentium delicatissimi an eos.<br>
+                 His expetenda reformidans ei, ea prompta iuvaret sapientem sea.
+                
+                    
+            </div>
+            <div class="projectText">
+                <p>Lorem ipsum dolor sit amet, probo idque postea ea qui,<br>
+                    diam petentium delicatissimi an eos.<br>
+                    His expetenda reformidans ei, ea prompta iuvaret sapientem sea.</p>
+                <p>Lorem ipsum dolor sit amet, probo idque postea ea qui,<br>
+                    diam petentium delicatissimi an eos.<br>
+                    His expetenda reformidans ei, ea prompta iuvaret sapientem sea.</p>
+                <p>Lorem ipsum dolor sit amet, probo idque postea ea qui,<br>
+                    diam petentium delicatissimi an eos.<br>
+                    His expetenda reformidans ei, ea prompta iuvaret sapientem sea.</p>
+
+            </div>
+            <div class= "interests2">
+                 <a href="portfolio.html">P1</a>
+                <img src=   "fruit1.webp" alt="Project 1"> 
```

Comment:
When naming images, go for more specific names and make sure that your alt tags are specific too - this is how people who need screen readers navigate your page!